### DISCUSSION TITLE
SteamVR Output

### DISCUSSION THREAD

**Discussion by TakkunTheCat** [freemocap/freemocap#99] [Ideas]
Demand for free or markerless full body tracking solutions is high. Given a way to mocap points live to SteamVR as trackers even as raw unsmoothed input would tap into a niche group that is passionate about making things like this work. More and more I see people buying mocapforall, or using juice's mediapipepose tracking purely to have tracking that's not attached to them, and they actively use, optimize their setups, and give feedback about the user experience and problems they encounter. It would be a great crowd to get feedback from even while there's yet to be a release version.

**Comment by RndUsername** [comment]
I thought about that, too! Would it be even possible to have the tracking be so fast, that it's live?
